core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -84,7 +88,7 @@
  * The default meta master.
  */
 @NotThreadSafe
-public final class DefaultMetaMaster extends CoreMaster implements MetaMaster, NoopJournaled {
+public final class DefaultMetaMaster extends CoreMaster implements MetaMaster {
```

Comment:
Should this implement `DelegatingJournaled`

Reply:
it already implements `Journaled`